gui/slick/js/new/config_backuprestore.js

```diff
@@ -1,5 +1,119 @@
-$(document).load(function(){
+$(document).ready(function(){
     $('#backupDir').fileBrowser({ title: 'Select backup folder to save to', key: 'backupPath' });
     $('#backupFile').fileBrowser({ title: 'Select backup files to restore', key: 'backupFile', includeFiles: 1 });
```

Comment:
Well either way it's an old bug and not something that's super important at the moment, open an issue and I'll look into it.